patternfly/patternfly-elements
# docs: Update testing docs with new examples

Updates docs to showcase the new testing framework.

### Preview

Link(s) to demo page(s) where this can be viewed:
- [Link](https://deploy-preview-1643--patternfly-elements.netlify.app/docs/develop/testing/) 


### Testing instructions

Verify https://deploy-preview-1643--patternfly-elements.netlify.app/docs/develop/testing/ makes sense.


### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
